packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -116,3 +116,80 @@ export interface CloudPulseMetricsList {
   metric: { [resourceName: string]: string };
   values: [number, string][];
 }
+
+export interface CreateAlertDefinitionPayload {
+  name: string | null;
```

Comment:
Why use null, we can use undefined which the default no value in typescript?

Reply:
@santoshp210 , any updates on this or any comments

Reply:
@venkymano-akamai  Yeah I'm making the changes